web/src/app/enterprise/sponsors/page.tsx

```
@@ -0,0 +1,427 @@
+      className="relative group h-full overflow-hidden rounded-xl p-px block"
+    >
+      <div className="relative z-10 h-full bg-linear-to-br from-[#161b22] to-[#0d1117] border border-[#30363d] rounded-xl p-6 transition-all duration-300 group-hover:border-transparent">
+        <div className="absolute inset-0 bg-linear-to-br from-[#fd7e14]/10 via-[#e8590c]/5 to-transparent opacity-0 group-hover:opacity-100 transition-opacity duration-500 rounded-xl"></div>
+        <div className="relative z-10 flex flex-col sm:flex-row sm:items-start sm:justify-between gap-4">
+          <div className="flex-1">
+            <h3 className="text-xl font-bold text-[#f0f6fc] mb-2">{tier.name}</h3>
+            <p className="text-gray-400 text-sm mb-3 group-hover:text-gray-300 transition-colors duration-300">{tier.description}</p>
+            <div className="space-y-2">
+              {tier.benefits.map((benefit, i) => (
+                <div key={i} className="flex items-start gap-2">
+                  <svg className="w-4 h-4 text-[#fd7e14] shrink-0 mt-0.5" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Biome (2.3.14)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)